core/schema/container.graphqls

```diff
@@ -27,10 +27,17 @@ type Container {
   build(context: DirectoryID!, dockerfile: String): Container!
 
   "This container's root filesystem. Mounts are not included."
-  fs: Directory!
+  rootfs: Directory!
```

Comment:
Let the bike shed commence.

I wasn't sure whether this should be `rootfs`, `rootFs` or `rootFS`.

This becomes `Rootfs()` and `WithRootfs()` in Go which is not ideal.

Which one works best in GraphQL and respective languages, @helderco @slumbering?

(note: if we were to call this `rootFs`, it would be converted as `rootFS` by Go codegen)

Reply:
In Python both `rootfs` and `root_fs` feel natural. I'm partial to `rootfs` but I see what you mean in Go.

Reply:
Thanks @helderco!

@sipsma @shykes Any thoughts on this?

- `rootfs` (graphql) ->`Rootfs()` (Go) -> `rootfs` (Python) -> ?? (Node)
- `rootFS` (graphql) -> `RootFS()` (Go) -> `root_fs` (Python) -> ?? (Node)

Reply:
> rootfs (graphql) ->Rootfs() (Go) -> rootfs (Python) -> ?? (Node)

That's my vote, I think it's pretty typical to write "rootfs"/"Rootfs" as a single word rather than emphasizing that "FS" is a acronym. Also just slightly easier to type.

Not a strong opinion though, either is fine with me.

Reply:
For typescript, based on: https://www.typescriptlang.org/tsconfig#watch-watchFile
I would say it would be either `rootfs` or `rootFs`.
I would prefer `rootfs`, but I'll let @slumbering have the last word :)

Reply:
We are using camelCase mostly. So I would go for:
rootfs (graphql) ->Rootfs() (Go) -> rootfs (Python) -> rootFs (so rootFS) (Node)